react/RemoveButton.tsx

```diff
@@ -1,21 +1,45 @@
-import React, { FunctionComponent } from 'react'
-import { IconDelete } from 'vtex.styleguide'
-import { useCssHandles } from 'vtex.css-handles'
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
 import { Loading } from 'vtex.render-runtime'
+import { useCssHandles } from 'vtex.css-handles'
+import { IconDelete, Button } from 'vtex.styleguide'
 
 import { useItemContext } from './ItemContext'
 import { opaque } from './utils/opaque'
 
 const CSS_HANDLES = ['removeButtonContainer', 'removeButton'] as const
 
-const RemoveButton: FunctionComponent = () => {
+type DisplayMode = 'icon-button' | 'text-button'
+type Variation =
+  | 'primary'
+  | 'secondary'
+  | 'tertiary'
+  | 'inverted-tertiary'
+  | 'danger'
+  | 'danger-tertiary'
+
+interface Props {
+  displayMode?: DisplayMode
+  variation?: Variation
+}
+
+function RemoveButton(props: Props) {
+  const { displayMode = 'text-button', variation = 'danger' } = props
```

Comment:
shouldn't the default be `icon-button` so we don't "break" existing stores?

Reply:
oh yeah, I was testing it and forgot to change